src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/Providers/AbstractSemanticQuickInfoProvider.cs

```diff
@@ -280,11 +280,10 @@ internal abstract partial class AbstractSemanticQuickInfoProvider : AbstractQuic
             SyntaxToken token,
             CancellationToken cancellationToken)
         {
+            var syntaxFacts = document.Project.LanguageServices.GetService<ISyntaxFactsService>();
```

Comment:
It seems `syntaxFacts` is only used at line 319 or below. Consider declaring around there, since there is possibility of returning early (without needing `syntaxFacts`).

Reply:
It turned out there was another call to `document.GetLanguageService<ISyntaxFactsService>()` just a few lines down. I changed it and left the `var syntactsFacts=` as is because there isn't any `return` between the initialization and the first use anymore.